### PR TITLE
Unify approve/lgtm experience in OTA repos

### PR DESCRIPTION
**Pull request by petr-muller**
Treat GitHub approval as `/lgtm` everywhere. Otherwise I'll *always* need to double tap PRs like https://github.com/openshift/cincinnati-graph-data/pull/3111#issuecomment-1430215775 :grimacing: 

/cc @wking @LalatenduMohanty @PratikMahajan

### CODE REVIEW

File: core-services/prow/02_config/openshift/cincinnati-operator/_pluginconfig.yaml
Comment (petr-muller) [inline core-services/prow/02_config/openshift/cincinnati-operator/_pluginconfig.yaml:5]: This is default so we may as well remove it

**Review by PratikMahajan — APPROVED** [review]
/lgtm

**Comment by petr-muller** [comment]
@LalatenduMohanty I assume https://github.com/openshift/cincinnati-graph-data/pull/3111#issuecomment-1430229161 was supposed to go here?

> What happens when PRs get auto approved tag? It is still need approved tag from an approver?

No, such PRs only need `lgtm` (by anyone in the org). It is assumed that approvers will only file PRs with desirable changes. But I think that's secondary in OpenShift. The concept of `approver` exists mainly for upstream to have a decision point about whether a certain change is desirable or not (the implementation quality is separate thing enforced by the `lgtm` label).

**Pull request merged**
